telos-frontend/src/components/todo/DashboardTodo.jsx

```diff
@@ -53,40 +53,40 @@ const outdated = {
 const DashboardTodo = () => {
   const listitems = [
     {
-      name: 'OnGoing',
+      name: 'isOverdue',
       due: '2021-04-30',
-      onGoing: true,
+      isOverdue: true,
       completed: false,
```

Comment:
I think onGoing and isOverdue have opposite meanings, is it okay to change the name to a contracting word without changing the state?

Reply:
The backend uses isOverdue, so I think we should stick with it to be consistent. I think isOverdue makes more sense than onGoing either way.
But yes, if you change from onGoing to isOverdue, you must switch the conditions around to keep everything the same.

Reply:
I see what you mean, I changed it to match JournalTodo, but I am not sure what the existing state names are now
 @ambiyu thoughts?